<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remote Diff Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_Path1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>File 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Path2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>File 2:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Path1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>431</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Path2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>431</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Compare">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Compare</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Log">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>511</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Browse1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Browse2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Path1_Local">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Local</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_File1</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_Path2_Local">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Local</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_File2</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_Path1_Remote">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Remote</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_File1</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_Path2_Remote">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Remote</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_File2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>552</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="menuConfig"/>
    <addaction name="menuAbout"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionItem_1">
   <property name="text">
    <string>Item 1</string>
   </property>
  </action>
  <action name="menuConfig">
   <property name="text">
    <string>Config</string>
   </property>
  </action>
  <action name="menuAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_File1"/>
  <buttongroup name="buttonGroup_File2"/>
 </buttongroups>
</ui>
